CHANGELOG.md

```diff
@@ -171,7 +194,8 @@ Initial release on crates.io
 - Working basic flash downloader with nRF51.
 - Introduce cargo-flash which can automatically build & flash the target elf file.
 
-[Unreleased]: https://github.com/probe-rs/probe-rs/compare/v0.6.2...master
+[Unreleased]: https://github.com/probe-rs/probe-rs/compare/v0.7.0...master
+[0.7.0]: https://github.com/probe-rs/probe-rs/releases/tag/v0.7.0
 [0.6.2]: https://github.com/probe-rs/probe-rs/releases/tag/v0.6.2
 [0.6.1]: https://github.com/probe-rs/probe-rs/releases/tag/v0.6.1
 [0.6.0]: https://github.com/probe-rs/probe-rs/releases/tag/v0.6.0
```

Comment:
Usually ranges are used for these as well, so the change log doesn't only show the differences between the tag and master but between the tags themselves.